source/features/show-associated-branch-prs-on-fork.css

```diff
@@ -0,0 +1,52 @@
+.pr-box {
+	display: flex;
+	-mox-box-align: center;
+	align-items: center;
+	height: 32px;
+}
+
+.pr-link {
+	color: rgb(125 133 144);
+	text-decoration: none;
+	font-weight: normal;
+	-mox-box-align: center;
+	align-items: center;
+}
+
+.pr-link .pr-label {
+	-moz-box-align: center;
+	align-items: center;
+	background-color: transparent;
+	border-width: 1px;
+	border-radius: 999px;
+	border-style: solid;
+	display: inline-flex;
+	font-weight: 600;
+	font-size: 12px;
+	line-height: 1;
+	white-space: nowrap;
+	border-color: rgb(48 54 61);
```

Comment:
GitHub already loads plenty of CSS variables, we should never inline colors, because they depend on the style. In this case the color is probably named something like `--color-border-muted`